gravityflow/gravityflowpdf
# Fix Custom name PDF Download via Merge Tag

## Description
Re: [Issue# 83](https://github.com/gravityflow/backlog/issues/83)
`{workflow_pdf_download_link}` merge tag does not download the custom file name PDF.

## Testing instructions
1. Create a PDF workflow step with a custom file name.
2. Create a Notification workflow step with the merge tag `{workflow_pdf_download_link}` sent via Message.
3. Process the workflow and confirm the PDF file generated and the one downloaded via merge tag are both correct with the custom file name.
4. Update the PDF step to follow the default file name.
5. Process the workflow again, and now the PDF generated and the one downloaded via merge tag would be following the default nomenclature.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
